src/app/qgssettingstree.cpp

```diff
@@ -251,7 +315,11 @@ QTreeWidgetItem *QgsSettingsTree::createItem( const QString &text,
     item = new QTreeWidgetItem( this, after );
 
   item->setText( 0, text );
-  item->setFlags( item->flags() | Qt::ItemIsEditable );
+  if ( !isGroup )
```

Comment:
Can you actually rename a group of settings? If so, it would be nice to keep the functionality.

Reply:
You can't rename anything at the moment -- the editable flag only works for the value. (This would be a nice feature though!)